api-reference/openapi_spec.json

```diff
@@ -8166,6 +8171,7 @@
       },
       "CustomerAcceptance": {
         "type": "object",
+        "description": "Passing this object during payments confirm . The customer_acceptance sub object is usually passed by the SDK or client",
```

Comment:
Please mention what to pass here

Reply:
Accepted, changing the description for this.